sample.env

```diff
@@ -23,3 +24,13 @@ MONGO_DB_PASS=
 MONGO_PORT=27017
 MONGO_URI=mongodb://${MONGO_DB_USER}:${MONGO_DB_PASS}@${MONGO_IP}:${MONGO_PORT}/
 MONGO_COLLECTION_TTL=7 # days
+
+# Destination to deposit messages to
+## Currently supported types are "s3" (Amazon S3 Bucket), "firehose" (Amazon Firehose), or "gcs" (Google Cloud Storage)
+DESTINATION=
+
+# Google Cloud Storage Variables
+## path to service account key file (json format)
+### If using gcs as a destination please note that the service account used must have
+### storage.buckets.get and storage.objects.create permissions to deposit messages correctly.
+GOOGLE_APPLICATION_CREDENTIALS=''
```

Comment:
It may be convenient for developers if this had the default value of the path to the example JSON file.

Reply:
Updated to use the default path